roles/grafana/tasks/main.yaml

```
@@ -60,3 +60,33 @@
     mode: '0640'
   notify: _tina_restart_grafana
   when: grafana__datasources_loki
+
+- name: Create dashboard provider
+  ansible.builtin.template:
+    src: dashboards.yaml.j2
+    dest: '{{ grafana__paths__provisioning }}/dashboards/dashboards.yaml'
+    owner: root
+    group: grafana
+    mode: '0640'
```

we don't need to make those private, they cannot contain secrets
```suggestion
    owner: root
    group: root
    mode: '0644'
```

---

I set this permissions because grafana sample file had 0640, but yes, no secrets here.